api/auth.go

```diff
@@ -461,6 +461,85 @@ func teamList(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 	return json.NewEncoder(w).Encode(result)
 }
 
+// title: team info
+// path: /teams/{name}
+// method: GET
+// produce: application/json
+// responses:
+//   200: Info team
+//   404: Not found
+//   401: Unauthorized
+func teamInfo(w http.ResponseWriter, r *http.Request, t auth.Token) error {
+	teamName := r.URL.Query().Get(":name")
+	team, err := auth.GetTeam(teamName)
+	if err != nil {
+		return err
+	}
+
+	_, err = t.Permissions()
+	if err != nil {
+		return err
+	}
+
+	apps, err := app.List(&app.Filter{})
```

Comment:
We should probably have a different list of apps that include only the ones owned by the team. You can use the `TeamOwner` field on the app.Filter.